cli/root.go

```diff
@@ -329,3 +361,30 @@ func FormatCobraError(err error, cmd *cobra.Command) string {
 	helpErrMsg := fmt.Sprintf("Run '%s --help' for usage.", cmd.CommandPath())
 	return cliui.Styles.Error.Render(err.Error() + "\n" + helpErrMsg)
 }
+
+func checkVersions(cmd *cobra.Command, client *codersdk.Client) error {
+	if varSuppressVersion {
+		return nil
+	}
+
+	clientVersion := buildinfo.Version()
+
+	info, err := client.BuildInfo(cmd.Context())
+	if err != nil {
+		return xerrors.Errorf("build info: %w", err)
+	}
+
+	fmtWarningText := `client/server versions do not match
+client version: %s
+server version: %s
+download the appropriate version from https://github.com/coder/coder/releases/tag/%s
```

Comment:
We could output the install script instead. Probably a bit simpler

Reply:
```
curl -L https://coder.com/install.sh | sh
```

Reply:
@kylecarbs doesn't that just install the latest CLI? That's not necessarily what a user would want. If the version is too new they're in the same situation.

Reply:
Nvm I saw the `--version` flag